spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_entities_spec.rb

```diff
@@ -433,7 +433,7 @@ module Requestors
             end
           end
 
-          context "applicant does not hae second property" do
+          describe "when the applicant does not hae second property" do
```

Comment:
```suggestion
          describe "when the applicant does not have second property" do
```

I know you've not changed this but there's a little typo